.gitignore

```diff
@@ -10,3 +10,4 @@
 package-lock.json
 yarn.lock
 venv/
+tooling/
```

Comment:
@shambergoldstein  per abel/instance validator instructions, a "tooling" virtual environment is created to work with these tools, and venv creates a separate folder under this name. If it's not added in gitignore it gets staged along with all extensions we make and we have to unstage it every time. "venv" is already in the gitignore and it's a standard name for the virtual environment, but since "tooling" is the name recommended by the tools readme, I thought it would justify its addition to gitignore.